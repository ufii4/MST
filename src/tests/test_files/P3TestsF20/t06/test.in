n 9
i 0;1;4.0
i 0;7;8.0
i 1;7;11.0
i 1;2;8.0
i 7;8;7.0
i 7;6;1.0
i 2;8;2.0
i 8;6;6.0
i 2;3;7.0
i 2;5;4.0
i 6;5;2.0
i 3;5;14.0
i 3;4;9.0
i 5;4;10.0
edge_count
mst
exit
