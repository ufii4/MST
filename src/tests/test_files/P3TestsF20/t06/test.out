success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
edge count is 14
mst 37.00
